activiti-core/activiti-api-impl/activiti-api-task-runtime-impl/src/main/java/org/activiti/runtime/api/event/impl/TaskCancelledImpl.java

```diff
@@ -23,16 +23,23 @@
 
 public class TaskCancelledImpl extends RuntimeEventImpl<Task, TaskRuntimeEvent.TaskEvents> implements TaskCancelledEvent {
 
+    private String reason;
+
     public TaskCancelledImpl() {
```

Comment:
is this constructor still useful?
if not, remove and make "reason" final

Reply:
True, we can get rid of that no-argument constructor